en/autogen/README.md

```diff
@@ -0,0 +1,68 @@
+# Autogeneration
+
+MAVSDK is available in a number of different languages.
+These all share the same "core" MAVLink implementation (written in C++), but almost all of the API-specific code for other languages is *autogenerated* from API definition files (and language-specific templates).
+
+This approach means that we don't have to maintain a separate MAVLink implementation or API for each language.
+New features are implemented (just once) in C++, and are then automatically available in Python, Mava, etc.
```

Comment:
Oh, what's this new language "Mava"? Should I learn it? Or is it a combination of Maven and Java? 😂